spec/unit/worker_spec.rb

```diff
@@ -165,12 +165,6 @@ def around_perform(job)
         worker.send(:log, :warn, 'my-message')
         expect(logger_io.string).to match(/my-message/)
       end
-
-      it 'can save log trace' do
```

Comment:
Should we not modify this test so that it still passes by creating a worker with a logger and a sighup handler that logs in a way to capture the expected output string?